Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractTouch.cs

```diff
@@ -299,6 +313,10 @@ private void CreateControllerRigidBody()
 
             var controllerRB = controllerRigidBodyObject.GetComponent<Rigidbody>();
 
+            controllerRigidBodyObject.name = string.Format("[{0}]_RigidBody_Holder", gameObject.name);
```

Comment:
I removed those 3 lines as I think that's what your branch looks like.
Done...but please make sure I did it right.